tests/init_test.py

```diff
@@ -0,0 +1,11 @@
+import unittest
```

Comment:
Id recommend using pytest is waaay less cumbersome then PyUnit.

Reply:
As I said in the conversation below its not a file that we created as part of this PR.
But in our future tests we sure will use pytest.